<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>taiko_key_binding</class>
 <widget class="QMainWindow" name="taiko_key_binding">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Taiko key setting</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="drum_l_list">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Drum Left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_back">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>570</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font-size: 22px;
	font: bold large &quot;Aerial&quot;;
    color:rgb(17,17,17);
	border-width: 1px;
	border-radius: 6px;
	border-bottom-color: rgb(150,150,150);
	border-right-color: rgb(165,165,165);
	border-left-color: rgb(165,165,165);
	border-top-color: rgb(180,180,180);
	border-style: solid;
	padding: 4px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}

QPushButton:hover {
	border-top-color: rgb(76,195,255);
	border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgb(76,195,255), stop:1 rgb(76,195,255));
	border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgb(76,195,255), stop:1 rgb(76,195,255));
	border-bottom-color: rgb(76,195,255);
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:pressed{
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QListWidget" name="drum_r_list">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add_drum_l">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font-size: 12px;
	font: bold large &quot;Aerial&quot;;
    color:rgb(17,17,17);
	border-width: 1px;
	border-radius: 6px;
	border-bottom-color: rgb(150,150,150);
	border-right-color: rgb(165,165,165);
	border-left-color: rgb(165,165,165);
	border-top-color: rgb(180,180,180);
	border-style: solid;
	padding: 4px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}

QPushButton:hover {
	border-top-color: rgb(76,195,255);
	border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgb(76,195,255), stop:1 rgb(76,195,255));
	border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgb(76,195,255), stop:1 rgb(76,195,255));
	border-bottom-color: rgb(76,195,255);
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:pressed{
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_remove_drum_l">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font-size: 12px;
	font: bold large &quot;Aerial&quot;;
    color:rgb(17,17,17);
	border-width: 1px;
	border-radius: 6px;
	border-bottom-color: rgb(150,150,150);
	border-right-color: rgb(165,165,165);
	border-left-color: rgb(165,165,165);
	border-top-color: rgb(180,180,180);
	border-style: solid;
	padding: 4px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}

QPushButton:hover {
	border-top-color: rgb(76,195,255);
	border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgb(76,195,255), stop:1 rgb(76,195,255));
	border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgb(76,195,255), stop:1 rgb(76,195,255));
	border-bottom-color: rgb(76,195,255);
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:pressed{
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QListWidget" name="rim_l_list">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="rim_r_list">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>330</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add_rim_l">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font-size: 12px;
	font: bold large &quot;Aerial&quot;;
    color:rgb(17,17,17);
	border-width: 1px;
	border-radius: 6px;
	border-bottom-color: rgb(150,150,150);
	border-right-color: rgb(165,165,165);
	border-left-color: rgb(165,165,165);
	border-top-color: rgb(180,180,180);
	border-style: solid;
	padding: 4px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}

QPushButton:hover {
	border-top-color: rgb(76,195,255);
	border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgb(76,195,255), stop:1 rgb(76,195,255));
	border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgb(76,195,255), stop:1 rgb(76,195,255));
	border-bottom-color: rgb(76,195,255);
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:pressed{
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_remove_rim_l">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font-size: 12px;
	font: bold large &quot;Aerial&quot;;
    color:rgb(17,17,17);
	border-width: 1px;
	border-radius: 6px;
	border-bottom-color: rgb(150,150,150);
	border-right-color: rgb(165,165,165);
	border-left-color: rgb(165,165,165);
	border-top-color: rgb(180,180,180);
	border-style: solid;
	padding: 4px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}

QPushButton:hover {
	border-top-color: rgb(76,195,255);
	border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgb(76,195,255), stop:1 rgb(76,195,255));
	border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgb(76,195,255), stop:1 rgb(76,195,255));
	border-bottom-color: rgb(76,195,255);
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:pressed{
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add_drum_r">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font-size: 12px;
	font: bold large &quot;Aerial&quot;;
    color:rgb(17,17,17);
	border-width: 1px;
	border-radius: 6px;
	border-bottom-color: rgb(150,150,150);
	border-right-color: rgb(165,165,165);
	border-left-color: rgb(165,165,165);
	border-top-color: rgb(180,180,180);
	border-style: solid;
	padding: 4px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}

QPushButton:hover {
	border-top-color: rgb(76,195,255);
	border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgb(76,195,255), stop:1 rgb(76,195,255));
	border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgb(76,195,255), stop:1 rgb(76,195,255));
	border-bottom-color: rgb(76,195,255);
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:pressed{
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_remove_drum_r">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font-size: 12px;
	font: bold large &quot;Aerial&quot;;
    color:rgb(17,17,17);
	border-width: 1px;
	border-radius: 6px;
	border-bottom-color: rgb(150,150,150);
	border-right-color: rgb(165,165,165);
	border-left-color: rgb(165,165,165);
	border-top-color: rgb(180,180,180);
	border-style: solid;
	padding: 4px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}

QPushButton:hover {
	border-top-color: rgb(76,195,255);
	border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgb(76,195,255), stop:1 rgb(76,195,255));
	border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgb(76,195,255), stop:1 rgb(76,195,255));
	border-bottom-color: rgb(76,195,255);
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:pressed{
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add_rim_r">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font-size: 12px;
	font: bold large &quot;Aerial&quot;;
    color:rgb(17,17,17);
	border-width: 1px;
	border-radius: 6px;
	border-bottom-color: rgb(150,150,150);
	border-right-color: rgb(165,165,165);
	border-left-color: rgb(165,165,165);
	border-top-color: rgb(180,180,180);
	border-style: solid;
	padding: 4px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}

QPushButton:hover {
	border-top-color: rgb(76,195,255);
	border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgb(76,195,255), stop:1 rgb(76,195,255));
	border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgb(76,195,255), stop:1 rgb(76,195,255));
	border-bottom-color: rgb(76,195,255);
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:pressed{
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_remove_rim_r">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font-size: 12px;
	font: bold large &quot;Aerial&quot;;
    color:rgb(17,17,17);
	border-width: 1px;
	border-radius: 6px;
	border-bottom-color: rgb(150,150,150);
	border-right-color: rgb(165,165,165);
	border-left-color: rgb(165,165,165);
	border-top-color: rgb(180,180,180);
	border-style: solid;
	padding: 4px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}

QPushButton:hover {
	border-top-color: rgb(76,195,255);
	border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgb(76,195,255), stop:1 rgb(76,195,255));
	border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgb(76,195,255), stop:1 rgb(76,195,255));
	border-bottom-color: rgb(76,195,255);
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:pressed{
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Drum Right</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rim Left</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>290</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rim Right</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="smoke_screen">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>681</width>
      <height>801</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGraphicsView{
background-color: rgba(0, 0, 0, 0.8);
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_ask_key">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>471</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
color : white;
}</string>
    </property>
    <property name="text">
     <string>Please press a key
(Press escape to cancle)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>676</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
